src/emc/plots/spatial.py

```diff
@@ -0,0 +1,136 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.colors import Normalize
+from matplotlib import rcParams, ticker, cm
+import cartopy.feature as cfeature
+import cartopy.crs as ccrs
+import matplotlib.colors as mcolors
+from cartopy.mpl.ticker import LongitudeFormatter, LatitudeFormatter
+
+
+def _global(latitude, longitude, data, **kwargs):
+    """
+    Plot data on global domain.
+    """
+    fig = plt.figure(figsize=(12, 10))
+    ax = fig.add_subplot(1, 1, 1,
+                         projection=ccrs.PlateCarree(central_longitude=0))
+    ax.add_feature(cfeature.GSHHSFeature(scale='auto'))
+    ax.set_extent([-180, 180, -90, 90])
+    ax.set_xticks([-180, -120, -60, 0, 60, 120, 180], crs=ccrs.PlateCarree())
+    ax.set_yticks([-90, -60, -30, 0, 30, 60, 90], crs=ccrs.PlateCarree())
+    lon_formatter = LongitudeFormatter(zero_direction_label=False)
+    lat_formatter = LatitudeFormatter()
+    ax.xaxis.set_major_formatter(lon_formatter)
+    ax.yaxis.set_major_formatter(lat_formatter)
+
```

Comment:
```suggestion
vmax = plotopts.get('vmax', data.max())
vmax = plotopts.get('vmax', data.min())

```

Reply:
@kevindougherty-noaa I forgot about this syntax that @aerorahul pointed out, this is probably the way to go, it checks if the entry is in the dict, and if not, sets it to this value.